colossus-tests/src/test/scala/colossus/core/ServerSpec.scala

```diff
@@ -255,16 +254,9 @@ class ServerSpec extends ColossusSpec {
           withServer(server) {
             val c1 = TestClient(server.system, TEST_PORT)
             expectConnections(server, 1)
-            val c2 = TestClient(server.system, TEST_PORT, waitForConnected = false, connectRetry = NoRetry)
```

Comment:
Shouldn't this test still be maintaining that the Server rejects connections when it is at capacity?